### ISSUE TITLE
need a more comprehensive error when failing to launch

### ISSUE THREAD

**Issue by volodymyrss** [cta-epfl/batchspawner#34]
right now it does not launch and it's unclear why

![image](https://github.com/cta-epfl/batchspawner/assets/3909535/f136a61f-cfa2-405d-b343-6e40916edde7)

**Comment by Ph0tonic** [comment]
So, what we want is to differentiate 2 cases:

1. When there is no certificate or that the certificate is outdated
2. When arc resources are unavailable

Is it correct ?

**Comment by volodymyrss** [comment]
Yes, though there could be various other details in case of ARC failure. Crucially, we should issue a comprehensive error to the user after parsing the arcsub message. If we can not parse arcsub message, we should send ourself a sentry alert and tell user its an internal error and we are working on it.
